components/StarSearch/StarSearchButton.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import { ButtonProps } from "react-day-picker";
+import Tooltip from "components/atoms/Tooltip/tooltip";
+
+interface StarSearchButtonProps {
+  onOpen: ButtonProps["onClick"];
+}
+
+export const StarSearchButton = ({ onOpen }: StarSearchButtonProps) => {
+  return (
+    <Tooltip
+      direction="top"
+      defaultOpen={true}
+      className="text-center w-36"
+      content="Ask StarSearch about your Workspace"
+    >
+      <button className="p-2 rounded-full bg-gradient-to-br from-sauced-orange to-amber-400 w-max" onClick={onOpen}>
```

Comment:
@isabensusan, I couldn't find this in Figma, so the orange shadow that you have is not present. Happy to update the component, but just getting this up to get the ball rolling.

![image](https://github.com/open-sauced/app/assets/833231/e1e0eacb-99b5-4275-91f0-aa7e87189412)